test/testContext.js

```diff
@@ -15,7 +15,7 @@ let survey = null
  * before executing all tests
  */
 const initTestContext = async () => {
-  user = await UserService.fetchUserByCognitoUsername('demo_user')
+  user = await UserService.fetchUserByUuid('demo_user')
 }
```

Comment:
Is the uuid demo_user?

Reply:
I don't think so...! That was the username...

Reply:
so are the tests working??

Reply:
Fixed, they work now